server/src/com/thoughtworks/go/server/service/UserService.java

```diff
@@ -203,6 +203,10 @@ public void modifyRolesAndUserAdminPrivileges(final List<String> users, final Tr
         return roles;
     }
 
+    public Collection<String> allRoleNames() {
+        return allRoleNames(goConfigService.cruiseConfig());
+    }
+
     public Collection<Role> allRoles(CruiseConfig cruiseConfig) {
```

Comment:
Not correct, this is used in the controller to list all roles for autocomplete in the authorizations form fields